sites/game-client/body-mobile.ejs

```diff
@@ -61,7 +61,11 @@
           <canvas id="inputcanvas"></canvas>
           <div id="joystick-container">
             <!-- TODO: styling -->
-            <div id="joystick-control-point"></div>
+            <div id="joystick-control-point-container">
+              <div id="joystick-control-point"></div>
+            </div>
+            <div id="joystick-ghost-mode-button" class="container-center hoverable"><img src="/static/sites/game-client/input-manager/ghostmode.png"></div>
+            <div id="joystick-game-button" class="container-center hoverable"><img src="/static/sites/game-client/input-manager/bomb.png"></div>
```

Comment:
Not really sure if we should make this part of the bombman extension, as in, if we remove the bombman extension, the button will disappear. If it takes too much time, we can leave a TODO here instead.

Reply:
The "game button" would be used by any extension which needs a button. It is not limited to bombman extension. For example, battle royale may need to use this button as well.
It requires too much engineering to allow extensions to add buttons. Let it be TODO would be good for now.